### PR TITLE
Updated docstrings to Google style format

### PR DESCRIPTION
**Pull request by suparnasnair**
# Updated docstrings for two methods _install_gist_module() and _install_module_dependencies() to Google style format docstrings.

_Dependencies that are required for this change: None_

Fixes #532 

## Status
**READY** 

## Type of change
- Documentation (fix or adds documentation)

### CODE REVIEW

File: opsdroid/loader.py [hunk 1]
```diff
@@ -536,7 +536,7 @@ def _install_module(self, config):
         Args:
             self: instance method
             config: dict of module config fields
-
+
```
Comment (FabioRosado) [inline opsdroid/loader.py]: Linting is failing due to the white space on this blank line. Could you remove it please?

File: opsdroid/loader.py [hunk 2]
```diff
@@ -588,6 +588,16 @@ def _is_gist_module(config):
         return "gist" in config
 
     def _install_module_dependencies(self, config):
+        """Install the dependencies of the module.
+
```
Comment (FabioRosado) [inline opsdroid/loader.py]: Same thing here, this line has white spaces could you remove them but keep the blank line?

File: opsdroid/loader.py [hunk 3]
```diff
@@ -588,6 +588,16 @@ def _is_gist_module(config):
         return "gist" in config
 
     def _install_module_dependencies(self, config):
+        """Install the dependencies of the module.
+        
+        Args:
+            self: instance method
+            config: dict of the module config fields
```
Comment (FabioRosado) [inline opsdroid/loader.py]: You have a blank space at the end of `config fields` can you remove it?

File: opsdroid/loader.py [hunk 4]
```diff
@@ -588,6 +588,16 @@ def _is_gist_module(config):
         return "gist" in config
 
     def _install_module_dependencies(self, config):
+        """Install the dependencies of the module.
+        
+        Args:
+            self: instance method
+            config: dict of the module config fields 
+
```
Comment (FabioRosado) [inline opsdroid/loader.py]: Another white space

File: opsdroid/loader.py [hunk 5]
```diff
@@ -588,6 +588,16 @@ def _is_gist_module(config):
...
 
     def _install_module_dependencies(self, config):
+        """Install the dependencies of the module.
+        
+        Args:
+            self: instance method
+            config: dict of the module config fields 
+        
+        Returns:
+            bool: True if installation succeeds
```
Comment (FabioRosado) [inline opsdroid/loader.py]: White space after succeeds, you need to remove it

File: opsdroid/loader.py [hunk 6]
```diff
@@ -664,6 +674,13 @@ def _install_local_module(config):
             _LOGGER.error("Failed to install from %s", str(config["path"]))
 
     def _install_gist_module(self, config):
+        """Install a module from gist path.
+
```
Comment (FabioRosado) [inline opsdroid/loader.py]: White space on blank page

File: opsdroid/loader.py [hunk 7]
```diff
@@ -664,6 +674,13 @@ def _install_local_module(config):
             _LOGGER.error("Failed to install from %s", str(config["path"]))
 
     def _install_gist_module(self, config):
+        """Install a module from gist path.
+        
+        Args:
+            self: instance method
+            config: dict of module config fields
+
```
Comment (FabioRosado) [inline opsdroid/loader.py]: white space on blank page

**Comment by suparnasnair** [comment]
Sure, I'm working on it. Thanks!

**Comment by FabioRosado** [comment]
This is awesome, thank you for fixing the issue will merge the PR soon.

We love to thank contributors by sending them [opsdroid stickers](https://medium.com/opsdroid/contributor-sticker-packs-738058ceda59)! Fill in the [claim form](https://forms.gle/iQZV9DjkaUiSaDXK6) to get yours!

**Comment by suparnasnair** [comment]
Thank you very much @FabioRosado!

**Pull request merged**